tests/conftest.py

```diff
@@ -33,6 +33,12 @@
 logging.getLogger('').setLevel(logging.INFO)
 
 
+@pytest.fixture(autouse=True)
+def run_around_tests():
+    yield
+    # assert Trade.use_db
```

Comment:
An empty fixture will not bring any benefit - so it's pointless code that needs executing - resulting in a slightly longer execution time.

this should either reset `use_db` (assuming that's the bug you're trying to solve) to ensure this is never in the wrong state after a test (backtesting is not the only part of the tests that can cause this to be left wrong if one test fails)

or the fixture should be removed.